src/main/java/teammates/common/util/Const.java

```diff
@@ -947,6 +947,7 @@ private Const() {
         public static final String LOGOUT = "/logout";
 
         public static final String DATABUNDLE = "/databundle";
+        public static final String DATABUNDLE_INDEX = "/databundle/index";
```

Comment:
It is more about search rather than index (index is a very technical term in database)